### ISSUE TITLE
Feature idea: Base64 alphabet variations

### ISSUE THREAD

**Issue by da411d** [javascript-obfuscator/javascript-obfuscator#647]
https://github.com/javascript-obfuscator/javascript-obfuscator/blob/master/src/custom-code-helpers/string-array/templates/string-array-calls-wrapper/AtobTemplate.ts#L9

To break base64 you can change base64 alphabets.
For example, original alphabet: 
`ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/=`

Variations:
```
abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789+/=
0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz+/=
0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ+/=
ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789/+=
ABCDEFGHIJKLMN0PQRSTUVWXYZabcdefghijklmnopqrstuvwxyzO123456789/+= // swap o and zero
ABCDEFGHlJKLMNOPQRSTUVWXYZabcdefghijkImnopqrstuvwxyz0123456789/+= // swap L and i
```

You can even just shuffle alphabet, but it will be very eazy to notice.

**Comment by sanex3339** [comment]
Really interesting idea. Will try it

**Comment by sanex3339** [comment]
The finest thing is that it covers both `base64` and `rc4` encodings.

**Comment by sanex3339** [comment]
https://github.com/javascript-obfuscator/javascript-obfuscator/pull/652

**Comment by sanex3339** [comment]
First implementation - only a single static alphabet variant.
Future - random variant between few alphabets that you described above.

**Comment by sanex3339** [comment]
Merged. Expect the new version soon

**Comment by sanex3339** [comment]
Released as `1.3.0`
